srtcore/buffer.cpp

```diff
@@ -1719,12 +1718,20 @@ int CRcvBuffer::readMsg(char* data, int len, ref_t<SRT_MSGCTRL> r_msgctl)
                 int64_t nowdiff = prev_now ? (nowtime - prev_now) : 0;
                 uint64_t srctimediff = prev_srctime ? (srctime - prev_srctime) : 0;
 
-                HLOGC(dlog.Debug, log << CONID() << "readMsg: DELIVERED seq=" << seq
-                        << " from POS=" << p << " T="
-                        << FormatTime(srctime) << " in " << (timediff/1000.0)
-                        << "ms - TIME-PREVIOUS: PKT: " << (srctimediff/1000.0)
-                        << " LOCAL: " << (nowdiff/1000.0)
-                        << " !" << BufferStamp(pkt.data(), pkt.size()));
+                int next_p = shift_forward(p);
+                CUnit* u = m_pUnit[next_p];
+                string next_playtime = "NONE";
+                if (u && u->m_iFlag == CUnit::GOOD)
+                {
+                    next_playtime = FormatTime(getPktTsbPdTime(u->m_Packet.getMsgTimeStamp()));
+                }
```

Comment:
I don't think this would be an improvement. The current statement says: This string should be "NONE", unless defined otherwise. Possibly `else` do some real improvement.